app/assets/javascripts/utils/fetch_utils.js

```diff
@@ -1,5 +1,5 @@
 import downloadjs from 'downloadjs';
-import Cookies from 'js-cookie';
+import * as Cookies from './cookies';
```

Comment:
I would rather use js-cookie than write our own cookie handler.

Reply:
js-cookie is an NPM module. The code is small enough that we shouldn't add another dependency.